app/utils/cu-profiling.ts

```diff
@@ -36,6 +226,11 @@ export function formatInstructionLogs({
 
     instructions.forEach((instruction, index) => {
         const programId = instruction.programId.toBase58();
+        const programName = getProgramName(programId, cluster);
+
+        const instructionName = getInstructionName(instruction as any);
```

Comment:
It looks like we could move this helper into @entities/cu-profiling. That is the place I would expect to find this logic